### DISCUSSION TITLE
日期选择器

### DISCUSSION THREAD

**Discussion by fdwszd123** [tusen-ai/naive-ui#5866] [Q&A]
日期选择器可以自定义类名吗？

**Comment by jahnli** [comment]
指的具体是哪一部分，如果没有 `api` 可以提个 `issue`

**Reply by fdwszd123** [reply]
就是我在使用日期选择器的时候，想自定义某些日期的类名来实现标记某些日期，antdv有提供dateRender插槽elementplus有cell-class-name我找了一圈naive貌似提供相关的api
